backend/benefit/applications/enums.py

```diff
@@ -255,6 +258,14 @@ class AhjoDecisionUpdateType(models.TextChoices):
     UPDATED = "Updated", _("Updated")
 
 
```

Comment:
Moved AhjoDecisionDetails here because otherwise it caused a circular import error.